test/patterns/import-cpp-bug-workaround/book.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    "pluginsConfig": {
+        "include-codeblock": {
+            "fixlang":"true"
```

Comment:
Yes.
 fixlang correct "c_cpp" to "cpp". By default, I set the policy to not fix the color, but let the user choose to fix it or not. (=> otherwise it conflicts with the ace plugin which use the aceMode "c_cpp")